src/main/java/com/easypost/service/EasyPostClient.java

```diff
@@ -7,8 +7,9 @@
 public class EasyPostClient {
     private final int connectTimeoutMilliseconds;
     private final int readTimeoutMilliseconds;
-    private final String apiBase = "v2";
     private final String apiKey;
+    private final String apiVersion = "v2";
+    private String apiBase;
```

Comment:
Make this final, since it won't change after being set by the constructor.

Reply:
I think this is just our personal preference, I like to use getter/setter for immutable variables in Java to follow the `encapsulation` practice, unless there is a significant advantage by making these two fields `public final` then I am down to make that change.

Reply:
Fair enough, won't block.